src/geo/map.js

```diff
@@ -230,25 +234,37 @@ cdb.geo.Layers = Backbone.Collection.extend({
    * the index should be recalculated
    */
   _assignIndexes: function(model, col, options) {
```

Comment:
Torque layers had a weight of 100. That meant that torque layers had orders like `101`, `102`, etc. This changes how orders are assigned so that layers are given sequential orders. The order of layers is now:

- Tiled Baselayer
- Mapnik layers
- (Torque layers)
- (Tiled layer with labels)